doc/devel/coding_style_guideline.rst

```diff
@@ -35,6 +35,15 @@ compliance of your code with PEP8. Beyond the aspects checked, as a
 contributor to DIPY, you should try to ensure that your code, including
 comments, conform to the above principles.
 
+Imports
+-------
+DIPY recommends using the following package shorthands to increase consistency
+and readability across the library::
+
+  import nibabel as nib
```

Comment:
If we prefer using `nb`, or rather than using a per-package rule we'd like the shorthands to be composed by two characters, I'm open to suggestions.